test/integration/src2srcmap.spec.ts

```diff
@@ -173,13 +172,11 @@ describe("Src2src map test", () => {
                     if (instrNodes === undefined) {
```

Comment:
We can potentially unnset the if and have a top-level `assert(instrNodes !== undefined, ...)`

Reply:
Addressed in https://github.com/ConsenSys/scribble/pull/94/commits/d4ed33eadbc26a96e9b7719cff33f967783af572